src/client/html/style.css

```diff
@@ -0,0 +1,777 @@
+@font-face {
+  font-family: Jaldi;
+  src: url(../assets/Jaldi-Regular.ttf);
+}
+
+@font-face {
+  font-family: Jaldi-Bold;
+  src: url(../assets/Jaldi-Regular.ttf);
+}
+
+:root {
+  --grey-lightest: #f7f9fa;
+  --white: #ffffff;
+  --grey: #c8d1dc;
+  --dark-grey: #6b7785;
+  --dark-blue: #1f2d3d;
+  --dark-blue-border: #2b3f56;
+  --green: #72cc18;
+  --blue-bright: #3988ff;
+  --blue-dark: #4673e9;
+  --black: #1f1f1f;
+  --grid-black-bg: rgba(0, 0, 0, 0.1);
+  --grid-red-bg: rgba(255, 0, 0, 0.1);
+  --red-bright: #ff0000;
+  --red-dark: #97150d;
+  --gold: #e3c421;
+  --peach: #ffeed5;
+  --font-family: Jaldi, "Helvetica Neue", Sans-Serif;
+  --font-family-headings: Jaldi-Bold, "Helvetica Neue", Sans-Serif;
+  --font-family-header: "Helvetica Neue", Sans-Serif;
+  --grid-bg: linear-gradient(to right, var(--grid-red-bg) 1px, transparent 1px),
+    linear-gradient(to bottom, var(--grid-red-bg) 1px, transparent 1px),
+    linear-gradient(to right, var(--grid-black-bg) 1px, transparent 1px),
+    linear-gradient(to bottom, var(--grid-black-bg) 1px, transparent 1px);
+  --grid-bg-size: 10px 10px, 10px 10px, 5px 5px, 5px 5px;
+  --dashed-border-bg: linear-gradient(90deg, var(--white) 50%, transparent 50%),
+    linear-gradient(90deg, var(--white) 50%, transparent 50%),
+    linear-gradient(0deg, var(--white) 50%, transparent 50%),
+    linear-gradient(0deg, var(--white) 50%, transparent 50%);
+  --dashed-border-bg-repeat: repeat-x, repeat-x, repeat-y, repeat-y;
+  --dashed-border-bg-size: 16px 2px, 16px 2px, 2px 16px, 2px 16px;
+  --dashed-border-bg-position: 0% 0%, 100% 100%, 0% 100%, 100% 0px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  display: flex;
+  width: 100%;
+  height: 100%;
+  font-family: var(--font-family);
+  margin: 0px;
+  padding: 0px;
+  background-color: var(--peach);
+}
+
+.header {
+  background-color: #fff;
+  min-width: 100%;
+  height: 52px;
+  position: fixed;
+  top: 0;
+  box-shadow: inset 0px -1px 0p var(--grey);
+  border-bottom: 1px solid var(--grey);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  z-index: 2;
```

Comment:
you shouldn't need the position:absolute, top, z-index here. If you make the container a flex column, it will make the header at the top and the game content right below. (I believe the `body` tag just needs a `flex-direction:column` property added and then you can remove all this. 

We typically want to have a `max-width` on the page content too (so the header and game content in this case) and then center is horizontally. This is because on super wide screens, it becomes hard to read the page content if it keeps growing horizontally. 
Example of no max-width and max-width:
![CleanShot 2022-06-03 at 15 40 08@2x](https://user-images.githubusercontent.com/18265845/171938542-5bdf796f-cdcd-4667-9ed1-b7335e4d3907.png)
![CleanShot 2022-06-03 at 15 40 45@2x](https://user-images.githubusercontent.com/18265845/171938545-e06d5eb1-841f-4c59-a7ea-c5a86106c2f9.png)
(would just need to add a solution for the leave button)

Reply:
On the flip side, I know it's not meant to be a mobile game, but I think we can make it just a tiny bit more mobile friendly. Fine in a followup PR, though. It can be a bit of a rabbit hole